src/components/Home/HomeFigure.tsx

```diff
@@ -33,11 +33,11 @@ export default function HomeFigure(props: Props) {
         alt="메인 이미지"
         onClick={() => props.handleSearch}
       />
-      {productLists?.map((productLists: IProductList, index: number) => {
+      {productLists?.map((IProduct, index) => {
```

Comment:
안에 value 는 타입 이름 말고 다른걸로 지정해주세요!

```suggestion
      {productLists?.map((product, index) => {
```

Reply:
넵 수정하겠습니다!